### ISSUE TITLE
Issues with some pages

### ISSUE THREAD

**Issue by BigNickBurgess** [duckduckgo/Android#235]
On the most recent version of DDG (5.0.2 50002) i'm having issues with certain sites.  The best example would be www.reddit.com which badly loads in mobile or when requesting desktop versions.

Other sites seem to load but then menu functions after logging in (which id imagine are Javascript controlled) dont work so you cant navigate the site.

Other sites still work fine in DDG so not sure what the issue is.  I've tested the same sites in Firefox and Firefox focus and all loads perfectly still in those.

**Comment by subsymbolic** [comment]
Thanks for reporting this @BigNickBurgess. I will investigate. A few questions to help me reproduce this:
• Which device and android version are you on?
• What is broken on reddit? Does the page load? Can you login?
• Does toggling off the privacy protection toggle in the Privacy Dashboard fix the issue? You can get there via the Privacy Grade button next to the search bar.

Thanks again for your help.

**Comment by BigNickBurgess** [comment]
Hi @subsymbolic ,

Samsung Galaxy S8 Oreo (CRB7)

Initial load on reddit just shows a visually broken version of their 'go to app'.

![screenshot_20180403-144420_duckduckgo](https://user-images.githubusercontent.com/31211392/38253073-a152d4b4-374d-11e8-9039-d98720bc9302.jpg)

Selecting request desktop site then shows a broken version of the site seemingly with no styling applied.

![screenshot_20180403-144438_duckduckgo](https://user-images.githubusercontent.com/31211392/38253099-b10dd6f6-374d-11e8-8015-a863bc37aceb.jpg)

As mentioned FF Focus with all filters toggled on still works fine and FF Android with uBlock, privacy badger, decentraleyes, cookie autodelete and HTTPS everywhere extensions all on works fine.

Yes toggling the privacy protection fixes it.

**Comment by subsymbolic** [comment]
Very useful, thanks @BigNickBurgess. I'll try and reproduce it here and get a patch out soon. My guess is that there is something wrong with the content blocking data persisted on your device. If you want to get going again before the patch is ready, a reinstall is likely to work.

**Comment by IOI-655321** [comment]
Yep cleared the Apps data and cache and its working again.  Thanks for looking into it.
